src/components/Tag.tsx

```diff
@@ -16,9 +18,14 @@ export function Tag<X extends Only<Xss<TagXss>, X>>({ text, type, xss, ...otherP
   const tid = useTestIds(otherProps);
 
   return (
-    <span {...tid} css={{ ...Css.dib.tinySb.ttu.px1.pyPx(4).gray900.br4.$, ...typeStyles, ...xss }} title={text}>
+    <span
+      {...tid}
+      css={{ ...Css.dif.tinySb.ttu.aic.gap(0.5).pxPx(6).pyPx(2).gray900.br4.$, ...typeStyles, ...xss }}
+      title={text}
+    >
       {/* Nesting `lineClamp` styles as the padding bottom set would expose the remainder of the text if applied on the same element */}
       {/* Using `lineClamp1` instead of `truncate` as `truncate` requires a width set to properly truncate and `lineClamp` can smartly do it based on the parent's width */}
+      {otherProps.icon && <Icon icon={otherProps.icon} inc={1.5} />}
```

Comment:
**suggestion:** When the text gets really long, then the icon is currently shrinking due to the flex container child having a default `flex-shrink: 1`, which is basically saying "Shrink your size as needed". 
<img width="219" alt="Screen Shot 2022-11-01 at 9 58 54 AM" src="https://user-images.githubusercontent.com/1143861/199250940-78ddfe6b-0f53-44d3-9968-5e17868c4ea1.png">

But we don't really want it to shrink here. What we can do is wrap this in another element and provide a `flex-shrink: 0`, telling it not to shrink. Something like:
```suggestion
      {otherProps.icon && <span css={Css.fs0.$}><Icon icon={otherProps.icon} inc={1.5} /></span>}
```

Reply:
Thanks for the explanation. It annoyed me, but I wasn't sure how to fix it.